### ISSUE TITLE
Error using summernote with bulma

### ISSUE THREAD

**Issue by ThomasTJdev** [ThomasTJdev/nim_websitecreator#37]
When editing pages with Summernote, bootstrap 4 is required due to the imported library. Check if BS4 is loaded else import normal summernote.

```html
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/summernote/0.8.10/summernote-bs4.css" integrity="sha256-fPUAOwSYkVTnL8xdLidCEi5IxW+ZVfcmNJ4m/+EGVI8=" crossorigin="anonymous" />
<script src="https://cdnjs.cloudflare.com/ajax/libs/summernote/0.8.10/summernote-bs4.js" integrity="sha256-D7wVIdz0dDoz21KQ/Z1cFnFXyrPcomZKBrVGLWE8vhg=" crossorigin="anonymous" defer></script>
```
https://github.com/ThomasTJdev/nim_websitecreator/blob/master/nimwcpkg/tmpl/utils.tmpl#L378-L379

**Comment by juancarlospaco** [comment]
Some day a plugin to add https://microsoft.github.io/monaco-editor would be nice, but it pulls NodeJS and half NPM.

**Comment by ThomasTJdev** [comment]
Fixed: ebdb25edc49d459b96c28b08ddd9f2fc660216bd

@juancarlospaco , open another tracker-issue for monaco?
